tasmota/language/de_DE.h

```diff
@@ -348,7 +348,7 @@
 #define D_MQTT_TLS_ENABLE "MQTT TLS"
 #define D_HTTP_API "HTTP API"
 #define D_HTTP_API_ENABLE "HTTP API aktivieren"
-#define D_FRIENDLY_NAME "Name [Friendly Name]"
+#define D_FRIENDLY_NAME "Friendly Name"
```

Comment:
Why not translate this one?

Reply:
Not everything should be translated as the are no better words in german. The previous expression breaks the two column design and doesn't say more than the new one.